src/style.css

```diff
@@ -0,0 +1,145 @@
+.html {
+  font-family: "Roboto", sans-serif;
+}
+
+.body {
+  margin: 0;
+}
+
+.card {
+  display: inline-flex;
+}
+
+.card__container {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  justify-content: center;
+  margin: 0;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  padding-top: 16px;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  flex-wrap: wrap;
+  width: 166px;
+  padding: 0 0 4px;
+  margin: 0;
+}
+
+.product {
+  display: inherit;
+  flex-direction: row;
+  justify-content: flex-start;
+}
+
+.product__text {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  padding: 0 0 16px;
+  color: #616070;
+}
+
+.rating__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.rating__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.rating__stars {
+  display: flex;
+}
+
+.rating__star:nth-child(-n + 4) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
Reuse stars block from previous task, it should have same functionality, if we change stars modifier it should define how many stars are active now